app/templates/brief.html

```diff
@@ -104,8 +104,8 @@
 <div class="grid-row">
   <div class="column-two-thirds">
     <header class="page-heading-smaller">
-      <p class="context">{{ brief.organisation }}</p>
-      <h1>{{ brief.title }}</h1>
+      <p class="govuk-caption-l">{{ brief.organisation }}</p>
```

Comment:
This should be a span?

```suggestion
      <span class="govuk-caption-l">{{ brief.organisation }}</p>
```

Reply:
Oh yes it should.

Thanks, fixed.